aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -1050,17 +1043,18 @@ where
 
     fn reads_needing_delayed_field_exchange(
         &self,
-    ) -> Vec<(<Self::Txn as Transaction>::Key, Arc<MoveTypeLayout>)> {
+    ) -> Vec<(
+        <Self::Txn as Transaction>::Key,
+        StateValueMetadata,
+        Arc<MoveTypeLayout>,
```

Comment:
Some of the above `reads_needing_delayed_field_exchange` functions have an additional u64 in output. Shouldn't this be having the same function signature?

Reply:
these are different traits. One is for transaction outputs, the other is for executor view